lib/energy_globe_awards/reprocess.rb

```diff
@@ -0,0 +1,80 @@
+require 'json'
+require 'text/hyphen'
+
+
+class Reprocess
+  VALID_CATEGORIES = %w(earth air water fire youth other)
+
+  def initialize source, destination
+    @source      = source
+    @destination = destination
+    @json        = nil
+  end
+
+  def reprocess
+    load_file
+    reverse
+    remove_descriptions
+    hyphenate
+    filter_categories
+    print_to_file
+  end
+
+  def load_file
+    raw   = File.read(@source)
+    @json = JSON.load(raw)
+  end
+
+  def reverse
+    @json = @json.reverse
+  end
+
+  def remove_descriptions
+    @json.map! { |item|
+      item.delete('description')
+      item
+    }
```

Comment:
`map`ping is not necessary, you could just `@json.each { |item| item.delete('description') }`